app/src/main/java/com/wpi/audiojournal/ui/theme/Theme.kt

```diff
@@ -21,21 +24,93 @@ private val LightColorPalette = lightColors(
     secondary = Salmon
 )
 
-data class ColorScheme(val colors: List<Color>, val background: Color, val content: Color, val borders: Color) {
+data class ColorScheme(val colors: List<Color>, val background: Color, val content: Color, val borders: Color, val infoBox: Color, val buttonText: Color) {
     fun getRainbowColor(index: Int) = colors[index % colors.size]
 }
 
 
 
-val LocalColorScheme = compositionLocalOf {
+
+
+
+val RainbowColorScheme = compositionLocalOf {
     ColorScheme(
         listOf(Purple, Salmon, Orange, Teal, Yellow, Blue),
         Gray,
         Color.Black,
+        Color.Black,
+        Color.White,
+        Color.Black
+    )
+}
+
+val BlueColorScheme = compositionLocalOf {
+    ColorScheme(
+        listOf(Color.White),
+        Blue,
+        Color.Black,
+        Color.Black,
+        Color.White,
         Color.Black
     )
 }
 
+val TealColorScheme = compositionLocalOf {
+    ColorScheme(
+        listOf(Color.White),
+        Teal,
+        Color.Black,
+        Color.Black,
+        Color.White,
+        Color.Black
+    )
+}
+
+val YellowColorScheme = compositionLocalOf {
+    ColorScheme(
+        listOf(Color.White),
+        Yellow,
+        Color.Black,
+        Color.Black,
+        Color.White,
+        Color.Black
+    )
+}
+
+val BrownColorScheme = compositionLocalOf {
+    ColorScheme(
+        listOf(Color.White),
+        Brown,
+        Color.Black,
+        Color.Black,
+        Color.White,
+        Color.Black
+    )
+}
+
+val BlackAndWhiteColorScheme = compositionLocalOf {
+    ColorScheme(
+        listOf(Color.Black),
+        Color.Black,
+        Color.Black,
+        Color.Black,
+        Color.White,
+        Color.White
+    )
+}
+
+val colorMap = hashMapOf<Int, ProvidableCompositionLocal<ColorScheme>>(
+    Pair(0, RainbowColorScheme),
+    Pair(1, BlueColorScheme),
+    Pair(2, TealColorScheme),
+    Pair(3, YellowColorScheme),
+    Pair(4, BrownColorScheme),
+    Pair(5, BlackAndWhiteColorScheme)
+)
+
+var schemeID = 0
+val LocalColorScheme = colorMap.get(schemeID)
```

Comment:
What is the purpose of doing this over a compositionLocal? compositionLocal allows us to dynamically modify the colorscheme at runtime using a provider, and automatically recomposes everything that uses it.